src/Benchmarks/Benchmarks.csproj

```diff
@@ -3,7 +3,8 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1;net451</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;net451</TargetFrameworks>
+    <RuntimeFrameworkVersion>2.0.0-*</RuntimeFrameworkVersion>
```

Comment:
Let me add some context the difference between common.props and dependencies.props.

The motivation for `dependencies.props` is to make updates easier through automation. Automatic version replacement on all csproj files was too unconstrained and often replaced versions for things it shouldn't have (like samples that are meant to be pinned). Constraining the automation to only files named dependnecies.props makes these errors less likely.

common.props, on the other hand, is not meant to be edited by automation. We don't always have a common.props file. The EF team, for example, prefers to call it "EntityFramework.props". MusicStore didn't need a common.props file.